RobotCode/src/main/java/frc/robot/Ports.java

```diff
@@ -0,0 +1,16 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.AnalogInput;
+
+public class Ports
+{
+	public static final int CLAW_LEFT_CLOSE = 0;
+    public static final int CLAW_LEFT_OPEN = 0;
+    public static final int PNEUMATICS_HUB = 0;
+    public static final int CLAW_RIGHT_OPEN = 0;
+    public static final int CLAW_RIGHT_CLOSE = 0;
+    public static final int ELEVATOR_LIMIT_SWITCH = 0;
+    public static final AnalogInput ELEVATOR_POTENTIOMETER = null;
+    public static int ELEVATOR_MOTOR_MAIN = 0;
```

Comment:
something is messed up here its either missing final designation or is incorrectly named